examples/kubernetes-kubenet-containerd.json

```diff
@@ -0,0 +1,41 @@
+{
+	"apiVersion": "vlabs",
+	"properties": {
+		"orchestratorProfile": {
+			"orchestratorType": "Kubernetes",
+			"orchestratorRelease": "1.13",
+			"kubernetesConfig": {
+				"networkPlugin": "kubenet",
+				"containerRuntime": "containerd",
+				"addons": [
+					{
+						"name": "kubernetes-dashboard",
```

Comment:
Is the dashboard literally non-functional in this cluster configuration?

Reply:
It's just copypasta. I'll change it to whatever you suggest.

Reply:
Cool, let's get rid of this addons config array altogether